runtime/src/lib.rs

```diff
@@ -301,14 +304,44 @@ impl pallet_balances::Config for Runtime {
 
 parameter_types! {
     pub const TransactionByteFee: Balance = 1;
+    pub MinimumMultiplier: Multiplier = Multiplier::saturating_from_integer(1);
+}
+
+/// Custom fee multiplier which remains constant regardless the network congestion
+/// TODO: consider using Substrate's built-in `pallet_transaction_payment::TargetedFeeAdjustment`
+/// to allow elastic fees based on network conditions (if that's more appropriate)
+pub struct ConstantFeeMultiplier<M>(sp_std::marker::PhantomData<M>);
+
+impl<M> MultiplierUpdate for ConstantFeeMultiplier<M>
+where
+    M: frame_support::traits::Get<Multiplier>,
+{
+    fn min() -> Multiplier {
+        M::get()
+    }
+    fn target() -> Perquintill {
+        Default::default()
+    }
+    fn variability() -> Multiplier {
+        Default::default()
+    }
+}
+impl<M> Convert<Multiplier, Multiplier> for ConstantFeeMultiplier<M>
+where
+    M: frame_support::traits::Get<Multiplier>,
+{
+    fn convert(previous: Multiplier) -> Multiplier {
+        let min_multiplier = M::get();
+        previous.max(min_multiplier)
+    }
```

Comment:
Any matching or comparing seems redundant while we providing really **Constant** FeeMultiplier.

```suggestion
    fn convert(_previous: Multiplier) -> Multiplier {
        M::get()
    }
```

Reply:
> Any matching or comparing seems redundant while we providing really **Constant** FeeMultiplier.

In fact, it should be the other way around:
```
fn convert(previous: Multiplier) -> Multiplier {
    previous
}
```
The reason we take max from 2 values is that currently the multiplier is 0, so unless we wipe the storage it will remain 0. Another option would be to set it to 1 in `on_runtime_upgrade()` hook. In either case in a subsequent PR we can modify it accordingly (once the pallet_transaction_payment::NextFeeMultiplier is successfully set to 1)